ci/env/install-dependencies.sh

```diff
@@ -95,10 +95,15 @@ install_miniconda() {
       msys*) miniconda_platform=Windows; exe_suffix=".exe";;
     esac
 
-    case "${OSTYPE}" in
-      # The hosttype variable is deprecated.
-      darwin*) HOSTTYPE="x86_64";;
-    esac
+    if [[ "${OSTYPE}" = darwin* ]]; then
```

Comment:
Can we do this block within the `case` statements above? really it seems like we should just have one case statement that sets all these variables for each platform.